msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md

```diff
@@ -49,7 +49,7 @@ The following image provides the flow to add authentication to external browsers
 
     ```JavaScript
        authentication.authenticate({
-          url: `${window.location.origin}/auth-start?oauthRedirectMethod={oauthRedirectMethod}&authId={authId}&hostRedirectUrl=${url}&googleId=${googleId}`,
+          url: `${window.location.origin}/auth-start?oauthRedirectMethod={oauthRedirectMethod}&authId={authId}&hostRedirectUrl={hostRedirectUrl}&googleId=${googleId}`,
```

Comment:
Is hostRedirectUrl is a place holder like authId or do we need to replace it with actual value?

Reply:
Yes it is a placeholder